applications/common/backend/src/main/resources/libs/kyanite/common/components/page/head-libs.html

```diff
@@ -18,13 +18,11 @@
 
 <sly data-sly-use.model="pl.ds.kyanite.common.components.helpers.PageModel"></sly>
 <!-- Google Tag Manager -->
-<script data-sly-test="${model.hasAnalyticsUrl}" type="text/plain" data-cookiecategory="analytics">
-  (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      '${model.googleAnalyticsScriptUrl @ context='uri'}?id='+i+dl;f.parentNode.insertBefore(j,f);
-  })(window,document,'script','dataLayer','${model.googleAnalyticsTrackingId @ context='text'}');
-</script>
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
Shouldn't we keep the `data-sly-test="${model.hasAnalyticsUrl}"` for the pages without GTM configuration set at all?

Reply:
You are right, we need that part, we only have to remove the `data-cookiecategory` property. I'll re-add that part.